File: java/Loop.java
import java.util.Scanner;

public class Loop {

    static void printArray(int[] dataArray) {
        for (int value: dataArray){
            System.out.printf("%d ",value);
        }
    }
    
    public static void main(String []args) {
        
        int size; // variable
        
        Scanner scanner = new Scanner(System.in);
        
        System.out.printf("Example your length of array in java :");
        size = scanner.nextInt();

        int array[] = new int[size]; // array

        // looping
        for (int i = 0; i < size; i++) {
            System.out.printf("input index [%d] value of array:", i);
            array[i] = scanner.nextInt();
        }

        printArray(array);
        System.out.println();

        for (int i = 0; i < size; i++) {
            System.out.printf("index [%s] value of array:%d ", i, array[i]);
            System.out.println();
        }

    }
}

File: javascript/loop.js
// function to print array in javascript
function printArray(array, length) {
    for (let n = 0; n < length; i++) {
     console.log("", array[n]);
    }
    console.log("\n");
}

// creating variable using var is not safety
// var array size, threshold = 100;
// trying uses another variables type likely let or const

const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
})

let size;
let array = [];

console.log("Example of Array in javascript \n");

readline.question("Enter length of size : ", (value) => {
    size = value;
    console.log("Your array size is ", value);
    
    for (let i = 0; i < size; i++) {
        readline.question("Input of index ",(input) => {
            array.push(input);
            readline.close();
        })
    }
})